jetson-xavier-seeed-hat.coffee

```diff
@@ -0,0 +1,44 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+
+BOARD_POWEROFF = 'Remove power from the Jetson Xavier NX Devkit SD-CARD and insert the freshly burnt sd-card.'
+BOARD_POWERON  = 'Power on the board.'
+
+module.exports =
+	version: 1
+	slug: 'jetson-xavier-seeed-hat'
+	aliases: [ 'jetson-xavier-seeed-hat' ]
+	name: 'Nvidia Jetson Xavier NX Devkit with seeed HAT'
```

Comment:
Also suggesting a more descriptive name like "Nvidia Jetson Xavier NX Devkit SD-CARD <Seeed HAT name>" or, if it has a short product code, even better.